src/features/farming/cakeStall/components/Cakes.tsx

```diff
@@ -75,13 +82,15 @@ export const Cakes: React.FC = () => {
               </span>
             </div>
           </div>
-          <Button
-            disabled={amount.lessThan(1)}
-            className="text-xs mt-1"
-            onClick={sell}
-          >
-            Sell
-          </Button>
+          {amount.gte(1) && (
+            <Button
+              disabled={amount.lessThan(1)}
```

Comment:
in what scenario will disable occur? this conflicts with `amount.gte(1)` render condition